packages/dashboard/src/components/Section/EditSections.styles.ts

```diff
@@ -0,0 +1,30 @@
+import { IEditSectionsStyles } from './Section.types';
+
+export const getStyles = (): IEditSectionsStyles => {
+  return {
+    root: {
+      width: '100%'
+    },
+    rightAlignedFlexContainer: {
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'flex-end',
+      width: '100%'
+    },
+    icon: {
+      color: 'black'
```

Comment:
>black [](start = 14, length = 5)

Can you please use the hash instead of black? I think that is what we are using through out fabric or use theme

Reply:
We really should use theme instead of hash or text. I will make it hash for now.

Reply:
We really should use theme instead of hash or text. I will make it hash for now. And in my next PR will grab the color from theme

---
In reply to: [214104856](https://github.com/OfficeDev/office-ui-fabric-react/pull/6153#discussion_r214104856) [](ancestors = 214104856)